### PR TITLE
etcdctlv3: better get command

### PR DESCRIPTION
**Pull request by xiang90**
/cc @gyuho @heyitsanthony

### CODE REVIEW

File: etcdctlv3/README.md [hunk 1]
```diff
@@ -42,6 +42,43 @@ Insert '--' for workaround:
 ./etcdctl put -- <key> <value>
 ```
 
+### GET [options] \<key\> [range_end]
+
+GET gets the key or a range of keys [key, range_end) if `range-end` is given.
+
+#### Options
+
```
Comment (heyitsanthony) [inline etcdctlv3/README.md]: alphabetize?

File: etcdctlv3/README.md [hunk 2]
```diff
@@ -42,6 +42,43 @@ Insert '--' for workaround:
...
+- order -- order of results; ASCEND or DESCEND
+
+- sort-by -- sort target; CREATE, KEY, MODIFY, VALUE, or VERSION
+
+- limit -- maximum number of results
+
+- hex -- print out key and value as hex encode string
+
```
Comment (heyitsanthony) [inline etcdctlv3/README.md]: TODO --prefix, --from

File: etcdctlv3/command/get_command.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	c := mustClientFromCmd(cmd)
+	kvapi := clientv3.NewKV(c)
+	resp, err := kvapi.Get(context.TODO(), key, opts...)
+	if err != nil {
+		ExitWithError(ExitError, err)
+	}
+
+	for _, kv := range resp.Kvs {
+		k, v := string(kv.Key), string(kv.Value)
+		if getHex {
```
Comment (heyitsanthony) [inline etcdctlv3/command/get_command.go]: not sure if `getHex` is necessary; why not use `%#v`?

**Comment by gyuho** [comment]
lgtm. defer to @heyitsanthony 

Thanks!

**Comment by xiang90** [comment]
PTAL @heyitsanthony

File: etcdctlv3/README.md (hunk 1 above)
Comment (xiang90) [inline etcdctlv3/README.md]: fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**